bigchaindb/consensus.py

```diff
@@ -31,8 +32,8 @@ def validate_transaction(bigchain, transaction):
             Descriptive exceptions indicating the reason the transaction failed.
             See the `exceptions` module for bigchain-native error classes.
         """
-        raise NotImplementedError
```

Comment:
Why do we remove this. Isn't this to make sure that you implement all methods when you sublclass this?